sdk/src/wallet/migration/migrate_0.rs

```diff
@@ -487,6 +514,16 @@ impl Convert for ConvertOutputMetadata {
     }
 }
 
+struct ConvertSegment;
+impl Convert for ConvertSegment {
+    type New = u32;
+    type Old = types::Crypto_0_18_0_Segment;
+
+    fn convert(old: Self::Old) -> crate::wallet::Result<Self::New> {
+        Ok(u32::from_be_bytes(old.bs))
```

Comment:
I guess this should use Self::New::from_be.. , but thats a very hard nit 🏃‍♂️

Reply:
extremely